app/jsx/layouts/DepartmentLayout.jsx

```diff
@@ -110,10 +131,7 @@ class DepartmentLayout extends React.Component {
             </section>
           </main>
           <aside>
-            <section className="o-columnbox1">
-              <ShareComp type="unit" id={this.props.unit.id} />
-            </section>
-            {this.props.sidebar}
+            {this.props.sidebar.props.data.length > 1 && this.props.sidebar}
```

Comment:
I don't think this is working. I know for instance that uclalaw has a sidebar, and it's not showing up now.